styleguide/components/21-atoms/inputs/input-multiselect-tagging/input-multiselect-tagging.bindings.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @file
+ * Javascript binding of baguettebox.min.js.
+ */
+'use strict';
+
+(function () {
+
```

Comment:
Best early return if Choices is undefined.

Reply:
Fixed.